vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.h

```diff
@@ -438,6 +438,10 @@ class LedgerImpl : public ledger::Ledger,
   scoped_refptr<base::SequencedTaskRunner> GetTaskRunner();
   void GetRewardsInternalsInfo(ledger::RewardsInternalsInfo* info) override;
 
+  void RefreshPublisher(
+      ledger::OnRefreshPublisherCallback callback,
```

Comment:
nit: can we do first publisher_key and then callback

Reply:
Done.